datafusion-cli/src/main.rs

```diff
@@ -51,6 +56,20 @@ pub async fn main() {
                 .validator(is_valid_batch_size)
                 .takes_value(true),
         )
+        .arg(
+            Arg::with_name("file")
+                .help("Execute commands from file, then exit")
+                .short("f")
+                .long("file")
+                .validator(is_valid_file)
+                .takes_value(true),
+        )
+        .arg(
```

Comment:
I can imagine other formats in the future other than CSV and pretty (for example, JSON using the Arrow json writer)

What would you think about making the option `--format=csv` rather than `--csv`, to leave room for future formats?

Reply:
I'm following `psql` convention. Deviation from that is okay but would be a thoughtful decision.

So far `psql` is a bit messy because you can have `csv`, `aligned` which is table, `unaligned` which is `|` separated values. In an ideal world we should strive for `csv`, `table`, `json`, `row-json` which is basically transposed json, and some more. WDYT?

Reply:
I would think at this moment we don't need to be compatible with `psql` per se regarding the command line arguments. So `--format=csv` would be a bit more user friendly I would say

Reply:
comment addressed